spec/openstates/models/bill_spec.rb

```diff
@@ -30,4 +30,20 @@
       end
     end
   end
+
+  describe ".find_by_openstates_id" do
+    context "openstates id is legit" do
+      it "should return a Bill" do
+        expect(OpenStates::Bill.find_by_openstates_id("KSB00005007")).to be_a described_class
```

Comment:
Ooooo, I forgot all about `described_class`. Such a wonderful tool. Thanks for the handy reminder :smile: